diff -Nur scribus-0.9.7/scribus/libabout/Makefile.am scribus-0.9.7-patch/scribus/libabout/Makefile.am
--- scribus-0.9.7/scribus/libabout/Makefile.am	Sat Feb  8 09:42:42 2003
+++ scribus-0.9.7-patch/scribus/libabout/Makefile.am	Fri Mar 14 14:34:19 2003
@@ -9,7 +9,7 @@
 LDFLAGS =  -s                    -export-dynamic 
 
 
-libabout_la_LDFLAGS = -version-info 0:0:0
+libabout_la_LDFLAGS = -avoid-version
 
 libabout_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libpdf/Makefile.am scribus-0.9.7-patch/scribus/libpdf/Makefile.am
--- scribus-0.9.7/scribus/libpdf/Makefile.am	Sat Feb  8 09:42:42 2003
+++ scribus-0.9.7-patch/scribus/libpdf/Makefile.am	Fri Mar 14 14:34:06 2003
@@ -10,7 +10,7 @@
 LDFLAGS =  -s                    -export-dynamic 
 
 
-libpdf_la_LDFLAGS = -version-info 0:0:0
+libpdf_la_LDFLAGS = -avoid-version
 
 libpdf_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libpostscript/Makefile.am scribus-0.9.7-patch/scribus/libpostscript/Makefile.am
--- scribus-0.9.7/scribus/libpostscript/Makefile.am	Sat Feb  8 09:42:42 2003
+++ scribus-0.9.7-patch/scribus/libpostscript/Makefile.am	Fri Mar 14 14:33:50 2003
@@ -10,7 +10,7 @@
 LDFLAGS =  -s                    -export-dynamic 
 
 
-libpostscript_la_LDFLAGS = -version-info 0:0:0
+libpostscript_la_LDFLAGS = -avoid-version
 
 libpostscript_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libprefs/Makefile.am scribus-0.9.7-patch/scribus/libprefs/Makefile.am
--- scribus-0.9.7/scribus/libprefs/Makefile.am	Sat Feb  8 09:42:42 2003
+++ scribus-0.9.7-patch/scribus/libprefs/Makefile.am	Fri Mar 14 14:34:32 2003
@@ -10,7 +10,7 @@
 LDFLAGS =  -s                    -export-dynamic 
 
 
-libpreferences_la_LDFLAGS = -version-info 0:0:0
+libpreferences_la_LDFLAGS = -avoid-version
 
 libpreferences_la_METASOURCES = AUTO
 
--- scribus-1.1.2/scribus/plugins/libchar/Makefile.am.orig	Thu Jun 19 12:53:49 2003
+++ scribus-1.1.2/scribus/plugins/libchar/Makefile.am	Tue Nov  4 11:47:35 2003
@@ -4,7 +4,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libcharselect_la_LDFLAGS = -version-info 0:0:0
+libcharselect_la_LDFLAGS = -avoid-version
 
 libcharselect_la_METASOURCES = AUTO
 
--- scribus-1.1.2/scribus/plugins/svgexplugin/Makefile.am.orig	Thu Jun 19 12:53:49 2003
+++ scribus-1.1.2/scribus/plugins/svgexplugin/Makefile.am	Tue Nov  4 12:05:50 2003
@@ -3,7 +3,7 @@
 plugins_LTLIBRARIES = libsvgexplugin.la
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 AM_LDFLAGS =  -s $(LIBFREETYPE_LIBS)      
-libsvgexplugin_la_LDFLAGS = -version-info 0:0:0
+libsvgexplugin_la_LDFLAGS = -avoid-version
 libsvgexplugin_la_METASOURCES = AUTO
 libsvgexplugin_la_SOURCES = svgexplugin.cpp 
 KDE_OPTIONS = qtonly
--- scribus-1.1.2/scribus/plugins/svgimplugin/Makefile.am.orig	Fri Aug 15 13:14:10 2003
+++ scribus-1.1.2/scribus/plugins/svgimplugin/Makefile.am	Tue Nov  4 12:06:09 2003
@@ -3,7 +3,7 @@
 plugins_LTLIBRARIES = libsvgimplugin.la
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 AM_LDFLAGS =  -s $(LIBFREETYPE_LIBS)      
-libsvgimplugin_la_LDFLAGS = -version-info 0:0:0
+libsvgimplugin_la_LDFLAGS = -avoid-version
 libsvgimplugin_la_METASOURCES = AUTO
 libsvgimplugin_la_SOURCES = svgplugin.cpp 
 KDE_OPTIONS = qtonly
--- scribus-1.1.2/scribus/plugins/printpreview/Makefile.am.orig	Thu Jun 19 12:53:49 2003
+++ scribus-1.1.2/scribus/plugins/printpreview/Makefile.am	Tue Nov  4 12:05:02 2003
@@ -4,7 +4,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libpreview_la_LDFLAGS = -version-info 0:0:0
+libpreview_la_LDFLAGS = -avoid-version
 
 libpreview_la_METASOURCES = AUTO
 
--- scribus-1.1.2/scribus/plugins/scriptplugin/Makefile.am.orig	Mon Oct 20 21:40:21 2003
+++ scribus-1.1.2/scribus/plugins/scriptplugin/Makefile.am	Tue Nov  4 12:05:24 2003
@@ -3,7 +3,7 @@
 AM_CPPFLAGS = -I$(top_builddir)/scribus $(PYTHONINC)
 plugins_LTLIBRARIES = libscriptplugin.la
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
-libscriptplugin_la_LDFLAGS = -version-info 0:0:0
+libscriptplugin_la_LDFLAGS = -avoid-version
 libscriptplugin_la_METASOURCES = AUTO
 libscriptplugin_la_SOURCES = cmdutil.cpp cmddialog.cpp cmddoc.cpp cmdpage.cpp cmdobj.cpp cmdgetprop.cpp cmdsetprop.cpp cmdtext.cpp cmdmani.cpp cmdcolor.cpp cmdmisc.cpp conswin.cpp pconsole.cpp scriptplugin.cpp 
 EXTRA_DIST = cmdvar.h cmdutil.cpp cmdutil.h cmddialog.cpp cmddialog.h cmddoc.cpp cmddoc.h cmdpage.cpp cmdpage.h cmdobj.cpp cmdobj.h cmdgetprop.cpp cmdgetprop.h cmdsetprop.cpp cmdsetprop.h cmdtext.cpp cmdtext.h cmdmani.cpp cmdmani.h cmdcolor.cpp cmdcolor.h cmdmisc.cpp cmdmisc.h scriptplugin.cpp scriptplugin.h pconsole.cpp pconsole.h conswin.cpp conswin.h 
