diff -ur scribus-1.2.5/scribus/libabout/Makefile.am scribus-1.2.5-nolibs/scribus/libabout/Makefile.am
--- scribus-1.2.5/scribus/libabout/Makefile.am	2006-03-29 21:28:17.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/libabout/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -3,7 +3,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libabout_la_LDFLAGS = -version-info 0:0:0
+libabout_la_LDFLAGS = -avoid-version
 
 libabout_la_METASOURCES = AUTO
 
diff -ur scribus-1.2.5/scribus/libpdf/Makefile.am scribus-1.2.5-nolibs/scribus/libpdf/Makefile.am
--- scribus-1.2.5/scribus/libpdf/Makefile.am	2006-03-29 21:28:17.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/libpdf/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -4,7 +4,7 @@
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
 
-libpdf_la_LDFLAGS = -version-info 0:0:0
+libpdf_la_LDFLAGS = -avoid-version
 
 libpdf_la_METASOURCES = AUTO
 
diff -ur scribus-1.2.5/scribus/libpostscript/Makefile.am scribus-1.2.5-nolibs/scribus/libpostscript/Makefile.am
--- scribus-1.2.5/scribus/libpostscript/Makefile.am	2006-03-29 21:28:18.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/libpostscript/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -3,7 +3,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libpostscript_la_LDFLAGS = -version-info 0:0:0
+libpostscript_la_LDFLAGS = -avoid-version
 
 libpostscript_la_METASOURCES = AUTO
 
diff -ur scribus-1.2.5/scribus/libprefs/Makefile.am scribus-1.2.5-nolibs/scribus/libprefs/Makefile.am
--- scribus-1.2.5/scribus/libprefs/Makefile.am	2006-03-29 21:28:18.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/libprefs/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -3,7 +3,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libpreferences_la_LDFLAGS = -version-info 0:0:0
+libpreferences_la_LDFLAGS = -avoid-version
 
 libpreferences_la_METASOURCES = AUTO
 
diff -ur scribus-1.2.5/scribus/plugins/fileloader/oodraw/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/fileloader/oodraw/Makefile.am
--- scribus-1.2.5/scribus/plugins/fileloader/oodraw/Makefile.am	2006-03-29 21:28:21.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/fileloader/oodraw/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-liboodrawimp_la_LDFLAGS = -version-info 0:0:0
+liboodrawimp_la_LDFLAGS = -avoid-version
 liboodrawimp_la_METASOURCES = AUTO
 liboodrawimp_la_SOURCES = oodrawimp.cpp stylestack.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/fontpreview/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/fontpreview/Makefile.am
--- scribus-1.2.5/scribus/plugins/fontpreview/Makefile.am	2006-03-29 21:28:21.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/fontpreview/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -4,7 +4,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libfontpreview_la_LDFLAGS = -version-info 0:0:0
+libfontpreview_la_LDFLAGS = -avoid-version
 libfontpreview_la_METASOURCES = AUTO
 libfontpreview_la_SOURCES = ui.cpp fontpreview.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/gettext/csvim/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/gettext/csvim/Makefile.am
--- scribus-1.2.5/scribus/plugins/gettext/csvim/Makefile.am	2006-03-29 21:28:22.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/gettext/csvim/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libcsvim_la_LDFLAGS = -version-info 0:0:0
+libcsvim_la_LDFLAGS = -avoid-version
 libcsvim_la_METASOURCES = AUTO
 libcsvim_la_SOURCES = csvdia.cpp csvim.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/gettext/docim/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/gettext/docim/Makefile.am
--- scribus-1.2.5/scribus/plugins/gettext/docim/Makefile.am	2006-04-09 22:04:02.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/gettext/docim/Makefile.am	2006-07-17 17:48:21.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libdocim_la_LDFLAGS = -version-info 0:0:0
+libdocim_la_LDFLAGS = -avoid-version
 libdocim_la_METASOURCES = AUTO
 libdocim_la_SOURCES = docim.cpp 
 
diff -ur scribus-1.2.5/scribus/plugins/gettext/htmlim/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/gettext/htmlim/Makefile.am
--- scribus-1.2.5/scribus/plugins/gettext/htmlim/Makefile.am	2006-03-29 21:28:22.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/gettext/htmlim/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(XML_CFLAGS) $(all_includes)
 
-libhtmlim_la_LDFLAGS = -version-info 0:0:0
+libhtmlim_la_LDFLAGS = -avoid-version
 libhtmlim_la_METASOURCES = AUTO
 libhtmlim_la_SOURCES = htmlim.cpp htmlreader.cpp 
 
diff -ur scribus-1.2.5/scribus/plugins/gettext/odtim/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/gettext/odtim/Makefile.am
--- scribus-1.2.5/scribus/plugins/gettext/odtim/Makefile.am	2006-03-29 21:28:25.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/gettext/odtim/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(XML_CFLAGS) $(all_includes)
 
-libodtimplugin_la_LDFLAGS = -version-info 0:0:0
+libodtimplugin_la_LDFLAGS = -avoid-version
 libodtimplugin_la_METASOURCES = AUTO
 libodtimplugin_la_SOURCES = odtim.cpp odtdia.cpp stylereader.cpp contentreader.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/gettext/sxwim/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/gettext/sxwim/Makefile.am
--- scribus-1.2.5/scribus/plugins/gettext/sxwim/Makefile.am	2006-03-29 21:28:26.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/gettext/sxwim/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(XML_CFLAGS) $(all_includes)
 
-libsxwimplugin_la_LDFLAGS = -version-info 0:0:0
+libsxwimplugin_la_LDFLAGS = -avoid-version
 libsxwimplugin_la_METASOURCES = AUTO
 libsxwimplugin_la_SOURCES = sxwim.cpp sxwdia.cpp stylereader.cpp contentreader.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/gettext/textfilter/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/gettext/textfilter/Makefile.am
--- scribus-1.2.5/scribus/plugins/gettext/textfilter/Makefile.am	2006-03-29 21:28:26.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/gettext/textfilter/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libtextfilterplugin_la_LDFLAGS = -version-info 0:0:0
+libtextfilterplugin_la_LDFLAGS = -avoid-version
 libtextfilterplugin_la_METASOURCES = AUTO
 libtextfilterplugin_la_SOURCES = textfilter.cpp tfdia.cpp tffilter.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/gettext/txtim/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/gettext/txtim/Makefile.am
--- scribus-1.2.5/scribus/plugins/gettext/txtim/Makefile.am	2006-03-29 21:28:27.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/gettext/txtim/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libtxtim_la_LDFLAGS = -version-info 0:0:0
+libtxtim_la_LDFLAGS = -avoid-version
 libtxtim_la_METASOURCES = AUTO
 libtxtim_la_SOURCES = txtim.cpp 
 
diff -ur scribus-1.2.5/scribus/plugins/libchar/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/libchar/Makefile.am
--- scribus-1.2.5/scribus/plugins/libchar/Makefile.am	2006-03-29 21:28:27.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/libchar/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -3,7 +3,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libcharselect_la_LDFLAGS = -version-info 0:0:0
+libcharselect_la_LDFLAGS = -avoid-version
 
 libcharselect_la_METASOURCES = AUTO
 
diff -ur scribus-1.2.5/scribus/plugins/newfromtemplateplugin/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/newfromtemplateplugin/Makefile.am
--- scribus-1.2.5/scribus/plugins/newfromtemplateplugin/Makefile.am	2006-03-29 21:28:28.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/newfromtemplateplugin/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libnewfromtemplateplugin_la_LDFLAGS = -version-info 0:0:0
+libnewfromtemplateplugin_la_LDFLAGS = -avoid-version
 libnewfromtemplateplugin_la_METASOURCES = AUTO
 libnewfromtemplateplugin_la_SOURCES = nftemplate.cpp nftdialog.cpp nftrcreader.cpp nftsettings.cpp nfttemplate.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/pixmapexport/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/pixmapexport/Makefile.am
--- scribus-1.2.5/scribus/plugins/pixmapexport/Makefile.am	2006-03-29 21:28:28.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/pixmapexport/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libscribusexportpixmap_la_LDFLAGS = -version-info 0:0:0
+libscribusexportpixmap_la_LDFLAGS = -avoid-version
 libscribusexportpixmap_la_METASOURCES = AUTO
 libscribusexportpixmap_la_SOURCES = export.cpp dialog.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/printpreview/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/printpreview/Makefile.am
--- scribus-1.2.5/scribus/plugins/printpreview/Makefile.am	2006-03-29 21:28:29.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/printpreview/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -3,7 +3,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libpreview_la_LDFLAGS = -version-info 0:0:0
+libpreview_la_LDFLAGS = -avoid-version
 
 libpreview_la_METASOURCES = AUTO
 
diff -ur scribus-1.2.5/scribus/plugins/psimport/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/psimport/Makefile.am
--- scribus-1.2.5/scribus/plugins/psimport/Makefile.am	2006-03-29 21:28:29.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/psimport/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libimportps_la_LDFLAGS = -version-info 0:0:0
+libimportps_la_LDFLAGS = -avoid-version
 libimportps_la_METASOURCES = AUTO
 libimportps_la_SOURCES = importps.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/saveastemplateplugin/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/saveastemplateplugin/Makefile.am
--- scribus-1.2.5/scribus/plugins/saveastemplateplugin/Makefile.am	2006-03-29 21:28:29.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/saveastemplateplugin/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libsaveastemplateplugin_la_LDFLAGS = -version-info 0:0:0
+libsaveastemplateplugin_la_LDFLAGS = -avoid-version
 libsaveastemplateplugin_la_METASOURCES = AUTO
 libsaveastemplateplugin_la_SOURCES = satemplate.cpp satdialog.cpp 
 
diff -ur scribus-1.2.5/scribus/plugins/scriptplugin/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/scriptplugin/Makefile.am
--- scribus-1.2.5/scribus/plugins/scriptplugin/Makefile.am	2006-03-29 21:28:29.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/scriptplugin/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -6,7 +6,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libscriptplugin_la_LDFLAGS = -version-info 0:0:0
+libscriptplugin_la_LDFLAGS = -avoid-version
 libscriptplugin_la_METASOURCES = AUTO
 libscriptplugin_la_SOURCES = valuedialog.cpp cmdutil.cpp cmddialog.cpp cmddoc.cpp cmdpage.cpp cmdobj.cpp cmdgetprop.cpp cmdsetprop.cpp cmdtext.cpp cmdmani.cpp cmdcolor.cpp cmdmisc.cpp objprinter.cpp objpdffile.cpp extmacro.cpp guiapp.cpp conswin.cpp pconsole.cpp scripterprefs.ui managemacrosdialog.ui editmacrodialog.ui macro.cpp macromanager.cpp scriptplugin.cpp objimageexport.cpp scriptercore.cpp
 
diff -ur scribus-1.2.5/scribus/plugins/short-words/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/short-words/Makefile.am
--- scribus-1.2.5/scribus/plugins/short-words/Makefile.am	2006-03-29 21:28:29.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/short-words/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -4,7 +4,7 @@
 
 INCLUDES = $(LIBFREETYPE_CFLAGS) $(all_includes)
 
-libscribusshortwords_la_LDFLAGS = -version-info 0:0:0
+libscribusshortwords_la_LDFLAGS = -avoid-version
 libscribusshortwords_la_METASOURCES = AUTO
 libscribusshortwords_la_SOURCES = vlnadialog.cpp parse.cpp shortwords.cpp configuration.cpp version.h
 
diff -ur scribus-1.2.5/scribus/plugins/svgexplugin/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/svgexplugin/Makefile.am
--- scribus-1.2.5/scribus/plugins/svgexplugin/Makefile.am	2006-03-29 21:28:30.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/svgexplugin/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -11,7 +11,7 @@
 endif
 AM_LDFLAGS =  $(linkflag) $(LIBFREETYPE_LIBS)
 
-libsvgexplugin_la_LDFLAGS = -version-info 0:0:0
+libsvgexplugin_la_LDFLAGS = -avoid-version
 libsvgexplugin_la_METASOURCES = AUTO
 libsvgexplugin_la_SOURCES = svgexplugin.cpp 
 
diff -ur scribus-1.2.5/scribus/plugins/svgimplugin/Makefile.am scribus-1.2.5-nolibs/scribus/plugins/svgimplugin/Makefile.am
--- scribus-1.2.5/scribus/plugins/svgimplugin/Makefile.am	2006-03-29 21:28:30.000000000 +0000
+++ scribus-1.2.5-nolibs/scribus/plugins/svgimplugin/Makefile.am	2006-07-17 17:47:00.000000000 +0000
@@ -12,7 +12,7 @@
 
 AM_LDFLAGS =  $(linkflag) $(LIBFREETYPE_LIBS)
 
-libsvgimplugin_la_LDFLAGS = -version-info 0:0:0
+libsvgimplugin_la_LDFLAGS = -avoid-version
 libsvgimplugin_la_METASOURCES = AUTO
 libsvgimplugin_la_SOURCES = svgplugin.cpp 
 
