diff -ur scribus-1.3.0/configure.in scribus-1.3.0-configure/configure.in
--- scribus-1.3.0/configure.in	2005-07-14 16:12:57.000000000 +0000
+++ scribus-1.3.0-configure/configure.in	2005-07-16 11:00:30.069234688 +0000
@@ -95,7 +95,7 @@
 # Usage: libmissing libfreetype required
 #        libmissing libfontconfig optional
 # to report a missing library
-function libmissing() {
+libmissing() {
 	# name of library
 	libname="${1}"
 	# one of "optional" or "required"
