--- scribus-1.3.3.11/CMakeLists.txt.orig	2008-05-11 15:01:48.000000000 +0000
+++ scribus-1.3.3.11/CMakeLists.txt	2008-05-11 15:06:05.000000000 +0000
@@ -74,6 +74,25 @@
     SET(ARCH_PPC 1)
 ENDIF (_machine_ppc)
 
+STRING(REGEX MATCH "(sparc-*)" _machine_sparc "${MACHINE}")
+IF (_machine_sparc)
+    MESSAGE(STATUS "Found target Sparc")
+    SET(ARCH_SPARC 1)
+ENDIF (_machine_sparc)
+
+STRING(REGEX MATCH "(sparcv9-*)" _machine_sparcv9 "${MACHINE}")
+IF (_machine_sparcv9)
+    MESSAGE(STATUS "Found target Sparc v9")
+    SET(ARCH_SPARCV9 1)
+ENDIF (_machine_sparcv9)
+
+STRING(REGEX MATCH "(sparc64-*)" _machine_sparc64 "${MACHINE}")
+IF (_machine_sparc64)
+    MESSAGE(STATUS "Found target Sparc64")
+    SET(ARCH_SPARC64 1)
+    SET(ARCH64BIT 1)
+ENDIF (_machine_sparc64)
+
 #convert any 64 bit build into generic 64 tag for below
 IF (ARCH_X86_64) 
   SET(ARCH64BIT 1)
