--- scribus-0.9.9/configure.in.orig	Sat Apr 19 12:09:44 2003
+++ scribus-0.9.9/configure.in	Wed May  7 12:09:44 2003
@@ -42,18 +42,18 @@
   fi
 fi
 
-AC_CHECK_LIB(socket,socket,
+AC_CHECK_FUNC(socket,,[AC_CHECK_LIB(socket,socket,
 if test "$uname" != "IRIX"; then
 	LIBS="-lsocket $LIBS"
 else
 	echo "Not using -lsocket since you are running IRIX."
-fi)
-AC_CHECK_LIB(nsl,gethostbyaddr,
+fi)])
+AC_CHECK_FUNC(gethostbyaddr,,[AC_CHECK_LIB(nsl,gethostbyaddr,
 if test "$uname" != "IRIX"; then
 	LIBS="-lnsl $LIBS"
 else
 	echo "Not using -lnsl since you are running IRIX."
-fi)
+fi)])
 LIBCUPS=""
 hacups=
 AC_CHECK_LIB(cups, httpConnect, [LIBCUPS="-lcups";hacups=yes],[LIBCUPS="";hacups=no])
