diff -Nur scribus-0.9.7/scribus/libabout/Makefile.am scribus-0.9.7-patch/scribus/libabout/Makefile.am
--- scribus-0.9.7/scribus/libabout/Makefile.am	Sat Feb  8 09:42:42 2003
+++ scribus-0.9.7-patch/scribus/libabout/Makefile.am	Fri Mar 14 14:34:19 2003
@@ -9,7 +9,7 @@
 LDFLAGS =  -s                    -export-dynamic 
 
 
-libabout_la_LDFLAGS = -version-info 0:0:0
+libabout_la_LDFLAGS = -avoid-version
 
 libabout_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libabout/Makefile.in scribus-0.9.7-patch/scribus/libabout/Makefile.in
--- scribus-0.9.7/scribus/libabout/Makefile.in	Sat Feb  8 14:11:04 2003
+++ scribus-0.9.7-patch/scribus/libabout/Makefile.in	Fri Mar 14 14:31:04 2003
@@ -235,7 +235,7 @@
 ####### kdevelop will overwrite this part!!! (end)############
 LDFLAGS = -s  
 
-libabout_la_LDFLAGS = -version-info 0:0:0
+libabout_la_LDFLAGS = -avoid-version
 
 #>- libabout_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libpdf/Makefile.am scribus-0.9.7-patch/scribus/libpdf/Makefile.am
--- scribus-0.9.7/scribus/libpdf/Makefile.am	Sat Feb  8 09:42:42 2003
+++ scribus-0.9.7-patch/scribus/libpdf/Makefile.am	Fri Mar 14 14:34:06 2003
@@ -10,7 +10,7 @@
 LDFLAGS =  -s                    -export-dynamic 
 
 
-libpdf_la_LDFLAGS = -version-info 0:0:0
+libpdf_la_LDFLAGS = -avoid-version
 
 libpdf_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libpdf/Makefile.in scribus-0.9.7-patch/scribus/libpdf/Makefile.in
--- scribus-0.9.7/scribus/libpdf/Makefile.in	Sat Feb  8 14:11:04 2003
+++ scribus-0.9.7-patch/scribus/libpdf/Makefile.in	Fri Mar 14 14:31:45 2003
@@ -236,7 +236,7 @@
 ####### kdevelop will overwrite this part!!! (end)############
 LDFLAGS = -s 
 
-libpdf_la_LDFLAGS = -version-info 0:0:0
+libpdf_la_LDFLAGS = -avoid-version
 
 #>- libpdf_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libpostscript/Makefile.am scribus-0.9.7-patch/scribus/libpostscript/Makefile.am
--- scribus-0.9.7/scribus/libpostscript/Makefile.am	Sat Feb  8 09:42:42 2003
+++ scribus-0.9.7-patch/scribus/libpostscript/Makefile.am	Fri Mar 14 14:33:50 2003
@@ -10,7 +10,7 @@
 LDFLAGS =  -s                    -export-dynamic 
 
 
-libpostscript_la_LDFLAGS = -version-info 0:0:0
+libpostscript_la_LDFLAGS = -avoid-version
 
 libpostscript_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libpostscript/Makefile.in scribus-0.9.7-patch/scribus/libpostscript/Makefile.in
--- scribus-0.9.7/scribus/libpostscript/Makefile.in	Sat Feb  8 14:11:04 2003
+++ scribus-0.9.7-patch/scribus/libpostscript/Makefile.in	Fri Mar 14 14:33:11 2003
@@ -236,7 +236,7 @@
 ####### kdevelop will overwrite this part!!! (end)############
 LDFLAGS = -s 
 
-libpostscript_la_LDFLAGS = -version-info 0:0:0
+libpostscript_la_LDFLAGS = -avoid-version
 
 #>- libpostscript_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libprefs/Makefile.am scribus-0.9.7-patch/scribus/libprefs/Makefile.am
--- scribus-0.9.7/scribus/libprefs/Makefile.am	Sat Feb  8 09:42:42 2003
+++ scribus-0.9.7-patch/scribus/libprefs/Makefile.am	Fri Mar 14 14:34:32 2003
@@ -10,7 +10,7 @@
 LDFLAGS =  -s                    -export-dynamic 
 
 
-libpreferences_la_LDFLAGS = -version-info 0:0:0
+libpreferences_la_LDFLAGS = -avoid-version
 
 libpreferences_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/libprefs/Makefile.in scribus-0.9.7-patch/scribus/libprefs/Makefile.in
--- scribus-0.9.7/scribus/libprefs/Makefile.in	Sat Feb  8 14:11:04 2003
+++ scribus-0.9.7-patch/scribus/libprefs/Makefile.in	Fri Mar 14 14:34:50 2003
@@ -236,7 +236,7 @@
 ####### kdevelop will overwrite this part!!! (end)############
 LDFLAGS = -s 
 
-libpreferences_la_LDFLAGS = -version-info 0:0:0
+libpreferences_la_LDFLAGS = -avoid-version
 
 #>- libpreferences_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/plugins/libchar/Makefile.am scribus-0.9.7-patch/scribus/plugins/libchar/Makefile.am
--- scribus-0.9.7/scribus/plugins/libchar/Makefile.am	Sat Feb  8 09:42:42 2003
+++ scribus-0.9.7-patch/scribus/plugins/libchar/Makefile.am	Fri Mar 14 14:35:35 2003
@@ -10,7 +10,7 @@
 LDFLAGS =  -s                    -export-dynamic 
 
 
-libcharselect_la_LDFLAGS = -version-info 0:0:0
+libcharselect_la_LDFLAGS = -avoid-version
 
 libcharselect_la_METASOURCES = AUTO
 
diff -Nur scribus-0.9.7/scribus/plugins/libchar/Makefile.in scribus-0.9.7-patch/scribus/plugins/libchar/Makefile.in
--- scribus-0.9.7/scribus/plugins/libchar/Makefile.in	Sat Feb  8 14:11:05 2003
+++ scribus-0.9.7-patch/scribus/plugins/libchar/Makefile.in	Fri Mar 14 14:35:52 2003
@@ -236,7 +236,7 @@
 ####### kdevelop will overwrite this part!!! (end)############
 LDFLAGS = -s 
 
-libcharselect_la_LDFLAGS = -version-info 0:0:0
+libcharselect_la_LDFLAGS = -avoid-version
 
 #>- libcharselect_la_METASOURCES = AUTO
 
